feat(content): Part 3 of Adding New Syndicate Business Missions (#11298)
endless-sky/endless-sky: data/human/human missions.txt

				goto youin
			label evilsweetner
			`	"Given your last foray into unregulated fuel sources, I know that you're looking for sky-high returns here. There's options here to cut through red tape and to make this facility a real profit multiplier.`
			label youin
			branch upcharge
				"syndicate business: mean to Howard" > 0
			`	"Now, the price for this investment is one hundred million credits for a robust 27,400 credits per day. And just like last time, no need to pull out your wallet - I can still pull a few strings with our friends at the bank and our Finance Division. For you, I've even talked them down to half interest. You'll just need to sign a few documents to get things moving."`
			`	He slides a document over to you, with a sleek pen ready for your signature. "Just sign this first document for the loan and then we can talk through the rest of the documents before signing."`
			choice
				`	(Sign the loan.)`
					goto nextynice
				`	"I'm done with these deals. I'm leaving."`

> 					decline

Author: Done as well!

			label upcharge
			`	Howard's tone carries a hint of impatience. "Now, the price for this investment is one hundred and eleven million credits for a 27,400 credit per day return. I've arranged a loan for you at the bank and through our Finance Division, so you won't need to come up with the investment immediately. I've talked down the loan's interest to be half as high as your prior loan. Just sign these documents to formalize the arrangement."`
			`	He shoves a document over to you, with a flimsy pen ready for your signature. "Just sign this first document for the loan and then we can talk through the rest of the documents before signing."`
			choice
				`	(Sign the loan.)`
					goto nextymean
				`	"I'm done with these deals. I'm leaving."`
					decline
			label nextynice
			action
				debt 100000000
					interest 0.04
					term 7300

> 			`	He takes the document from you and you see the loan terms on the screen behind him: 100M credits, 20 years, 15% rate.`

Author: Good catch, 15.7% but I just cut off the 0.7% instead of rounding.
```suggestion
			`	He takes the document from you and you see the loan terms on the screen behind him: 100M credits, 20 years, 16% rate.`
```

			choice
				`	"Sounds like a good plan to me."`
					goto choseplana
				`	"What would it take to boost profits even more?"`
					goto planb
					to display
						not "heardplanb"
				`	"And if I have concerns about making a positive impact?"`
					goto planc
					to display
						not "heardplanc"
				`	"Could you remind me about the more profitable option?"`
					goto planb
					to display
						has "heardplanb"
				`	"Explain the 'genuine good' choice once more."`
					goto planc
					to display
						has "heardplanc"
			label planb
			`	"There's a certain allure to uncompromising profit - a no-holds-barred, red-tape-free approach. We cut every corner on labor, safety, and the environment to unlock some explosive returns. It's the high-stakes, high-return option for those unafraid of a little controversy for rapid gains of 41,000 credits a day, or around 14% a year."`

> 			`	He grins. "And as for any little controversies? Don't worry about it - they didn't matter last time, and the odds of another cropping up with this project are probably slim."`

Reviewer: I think this flows better with the preceding line, but up to you.
```suggestion
			`	He grins. "And as for those little controversies? Don't worry about it - they didn't matter last time, and the odds of another cropping up with this project are probably slim."`
```

			label ending
			`	The long journey back to the spaceport proceeds in silence, leaving your decision behind to echo through the lives of the people left here.`

mission "Syndicate Business 6"
	name "Visit Howard on <planet>"
	description "Howard wants to tell you about yet another investment opportunity."
	minor
	to offer
		has "Syndicate Business 5: done"
		has "Syndicate Business 5.1 A 10m Expansion Update: offered"
		has "event: syndicate business 10m investment done"
		random < 100

> 		credits > 300000000

Author: ~~Just wait until you see the 1 billion mission!~~

My concern is that net worth might not be as immediately available, though if their net worth is higher than the three hundred million then they should have more than enough of a buffer.

So far, the pattern I've been using is

- 1M investment: 5M credits
- 10M investment: 40M credits
- 100M investment: 300M credits
- 1B investment: 2B credits

What if I add an `or` statement for 300M or 600M net worth?
```suggestion
		or
			credits > 300000000
			"net worth" > 600000000
```

> 		credits > 300000000

Reviewer 2: I think that works better, yes.

			label goldenjewelaround
			branch piratearound
				"syndicate business golden jewel dead"
			`	Wistfully, Howard adds, "Moktar. The Golden Jewel. I wonder if that ship's still flying out there...`
				goto askitself
			label piratearound
			`	Wistfully, Howard adds, "Moktar. The Golden Jewel. Almost a shame that pirate captain isn't flying around anymore...`
			label askitself
			`	"Anyway, back to the ask. Think you can plant evidence on a Quicksilver in <waypoints>, then make sure it turns into wreckage, and get to <planet> before <day>? Once you're on <planet>, I might need you to tie off any loose ends, but then right on through to that nine figure credit investment. What do you say?"`
			choice
				`	"I'm in, lets do it."`
					goto ending
				`	"How can I possibly plant evidence on a crewed ship without anyone knowing?"`
				`	"I'm out, we're through."`
					decline
			branch seendummy

> 				( "Syndicate target practice [0]: offered" + "Syndicate target practice [1]: offered" + "Syndicate target practice [2]: offered" + "Syndicate target practice [3]: offered" ) > 0

Author: Makes sense to me

				`	"If it isn't a pitiful, perplexing parasite, popping up to plague my peace once more!"`
				choice
					`	"We can't keep meeting like this."`
					`	"Uhhh... who are you again?"`
						goto whomst
				`	"Our paths are as persistently painful as the pestilence - your persistent presence has provoked my peevishness!"`
					goto power
				label whomst
				`	"I am the relentless, raging rapscallion who revels in ripping apart feeble, fumbling fiends!"`
				label power
				`	You can't help feeling a sense of deja vu as the Vanguard's engines roar to life and you see a half-baker's dozen fixed guns take aim at your ship, ready to strike.`
		on kill
			set "syndicate business golden jewel dead"
			dialog "The crew of the Golden Jewel are no more. You think you can make out the pirate captain drifting in space nearby."

	on enter "Oblate"
		"reputation: Test Dummy" = 10

	npc assist kill
		government "Test Dummy"
		personality disables staying heroic target derelict
			confusion 120
		system "Oblate"
		ship "Quicksilver Test Dummy" "Syndicate Test Vessel"
		on encounter
			conversation
				`You receive a message from Howard almost immediately upon entering the system:`

> 				`	"Alright, you're here. Get aboard the Quicksilver to upload the file, then make sure it doesn't leave the system in one piece, and finally meet back on <destination>. Remember, no loose ends here."`

Author: Hmmm, that's an interesting idea. I'm going to playtest with that to see if that would fit well or would be annoying to players.

			`		Howard Vendeur`
			`		Investor Relations Manager, Megaparsec Incorporated`

	on complete
		"reputation: Independent (Killable)" = 0
		"reputation: Test Dummy" = -1000
		conversation
			`As you maneuver your <model> onto the flat, ancient mesa that's considered spaceport parking, your ship's comms light up with a message:`
			branch meancanyonopening
				"syndicate business: mean to Howard" > 0
			`	"Excellent work, Captain. My Shuttle's waiting just outside the spaceport - please meet me there immediately. I appreciate your swift action on this.`
				goto restofcanyonopening
			label meancanyonopening
			`	"Good work. My Shuttle is parked just outside the spaceport - meet me there immediately so we can proceed without further delay. No time to waste.`
			label restofcanyonopening
			`		Howard Vendeur`
			`		Investor Relations Manager, Megaparsec Incorporated`

mission "Syndicate Business 8"
	name "Escort Howard to <planet>"
	description "Escort Howard to <destination> so that he can shore up support for his promotion."
	priority
	to offer
		has "Syndicate Business 7: done"
	source "Canyon"
	destination "Hephaestus"
	on offer
		conversation

> 			`You step out on the stone field surrounding Canyon's spaceport and quickly locate Howard's Shuttle in an open patch with a wide berth. Climbing aboard, you find Howard waiting with a measured nod.`

Author: I pictured that he's _been_ waiting, and then when you climb aboard he greets you with a nod. You're right that that the phrasing is weird. How's:
```suggestion
			`You step out on the stone field surrounding Canyon's spaceport and quickly locate Howard's Shuttle in an open patch with a wide berth. Climbing aboard, you find Howard waiting, and he greets you with a measured nod.`
```

> 			`You step out on the stone field surrounding Canyon's spaceport and quickly locate Howard's Shuttle in an open patch with a wide berth. Climbing aboard, you find Howard waiting with a measured nod.`

Reviewer 2: Much better.

	npc accompany save
		government "Syndicate"
		personality escort timid
		ship "Shuttle" "Gallant Knight"
		on kill
			dialog "You watch as Howard's Shuttle explodes into a cloud of debris along with any hope of his promotion."
	npc
		government "Bounty Hunter"
		personality harvests nemesis plunders staying
		system destination
		ship "Quicksilver" "Severance Package"
	npc
		government "Bounty Hunter"
		personality harvests nemesis plunders
		system destination
		ship "Quicksilver" "Hostile Takeover"

	on complete
		conversation
			`Your ship arrives amidst the sprawling metropolis of the Syndicate. You lose sight of Howard's Shuttle as you land your <model>, but nevertheless get a parting message from him:`
			`	"Captain <last>, I'll be in touch in a few days for the investment once some schmoozing has secured my promotion. Stay sharp.`
			`		Howard Vendeur`
			`		Investor Relations Manager, Megaparsec Incorporated`
			action
				event "syndicate business Howard schmoozing" 2 4

event "syndicate business Howard schmoozing"

mission "Syndicate Business 9"
	name "Meet Howard on <planet>"
	description "Meet Howard on <destination> to escort him back to Sunracer."
	landing
	to offer

> 		has "Syndicate Business 8: done"

Author: Fair, and that reminds me I should clear smoozing on offer so I can reuse it later.

		has "event: syndicate business Howard schmoozing"
	source
		near "Sol" 0 100
		not planet "Hephaestus"
	destination "Hephaestus"
	on offer

> 		conversation

Author: ```suggestion
		clear "event: syndicate business Howard schmoozing"
		conversation
```

			`After a few days, you get a triumphant message from Howard:`
			`	<first> <last>,`
			`	Promotion secured. Meet me here on <destination> to escort me back to Sunracer - I've got a new ship and office to show you. And of course plans to finalize our nine-figure credit investment.`
			`		Howard Vendeur`
			`		Director of Investor Relations & External Projects, Megaparsec Incorporated`
			choice
				`	(Message back a heartfelt congratulations and that you're on your way.)`
				`	(Respond that nobody cares about his promotion, but you'll be there soon)`
					goto rude

> 				`	(Ignore and delete the message.)`
> 					decline

Author: ```suggestion
```

			label verynice
			action
				"syndicate business: mean to Howard" --
			`	Howard's grin widens, his tone warm. "Glad you see it that way. I knew it'd catch your eye."`
				goto restofhello
			label verynotnice
			`	Howard's face darkens, hurt barely concealed as he snaps, "I thought you'd have better taste. But we have work to do."`
			action
				"syndicate business: mean to Howard" ++
			label restofhello
			`	Without missing a beat, Howard adds, "Enough talk. No time to waste, let's get to Sunracer." He strides inside his Bounder, and with a roar of engines he takes off for the journey ahead, with your ship behind him.`
				launch
	
	npc accompany save
		government "Syndicate"
		personality escort timid
		ship "Bounder" "Knight So Bold"
		on kill
			dialog "You watch as Howard's Bounder explodes into a cloud of debris along with any hope of that investment opportunity."
	npc
		government "Bounty Hunter"
		personality harvests nemesis plunders staying
		system destination
		ship "Quicksilver" "Enforced Compliance"
	npc
		government "Bounty Hunter"
		personality harvests nemesis plunders
		system destination
		ship "Quicksilver" "The Noncompete"
	on complete

> 		event "syndicate business 100m investment done" 730

Author: This is used as the mission being completed, but the check if you've actually chosen one of the investments is via conditions like `"Syndicate Business expansion 100m good"`. The main advantage is that this allows for time-delayed events/missions even if the player refused to invest (For example, with it here, Howard could send the player an email in 2 years saying he's richer than ever and that he's sure the player regrets missing out on this investment)

> 		event "syndicate business 100m investment done" 730

Reviewer 2: Ah, makes sense.

			choice
				`	"Sounds like a good plan to me."`
					goto choseplana
				`	"What would it take to boost profits even more?"`
					goto planb
					to display
						not "heardplanb"
				`	"And if I have concerns about making a positive impact?"`
					goto planc
					to display
						not "heardplanc"
				`	"Could you remind me about the more profitable option?"`
					goto planb
					to display
						has "heardplanb"
				`	"Explain the 'genuine good' choice once more."`
					goto planc
					to display
						has "heardplanc"
			label planb
			`	"There's a certain allure to uncompromising profit - a no-holds-barred, red-tape-free approach. We cut every corner on labor, safety, and the environment to unlock some explosive returns. It's the high-stakes, high-return option for those unafraid of a little controversy for rapid gains of 41,000 credits a day, or around 14% a year."`

> 			`	"And as for any little controversies? Don't worry about it - they didn't matter last time, and the odds of another cropping up with this project are probably slim."`

Author: Good catch

			`	"Howard leans forward with a calculated gleam in his eye. "Excellent choice - it's a high-octane approach where we push every boundary to extract maximum profit. Think of your high returns, while Megaparsec gets rapid product cycles and a clear market edge. Minor bumps in the PR and labor front are just part of the cost of being a true trailblazer."`
				goto ending
			label choseplanc
			action
				"salary: Syndicate Business" += 20400
				set "Syndicate Business expansion 100m good"
				log "Was able to make another investment with Megaparsec, this time one hundred million credits, after planting evidence on a Syndicate ship for Howard. Out of three investment options, ended up choosing the positive impact approach."
			`	Howard's expression softens as he speaks. "Well done - your choice speaks volumes about your commitment to ethical standards, sustainability, and worker well-being. While the monetary returns are unquestionably the worst offered to you, I'm sure it'll still build lasting value for Megaparsec through the more intangible benefits of enhanced community goodwill, reduced environmental liabilities, and robust employee loyalty."`
			label ending

> 			`	"Great, now that we've locked in your investment, let's get the remaining paperwork squared away." He slides a stack of documents across the desk. "These include the last of the remaining loan terms, the investment agreement for the testing facility, and the standard legal fine print. At this point, you know the drill - just a few signatures to seal the deal." He hands you a sleek pen, his gaze businesslike. "Once this is done, you'll be a stakeholder in one of Megaparsec's most innovative projects to redefine performance testing on Crossroads."`

Reviewer 2: ```suggestion
			`	He smiles. "Great, now that we've locked in your investment, let's get the remaining paperwork squared away." He slides a stack of documents across the desk. "These include the last of the remaining loan terms, the investment agreement for the testing facility, and the standard legal fine print. At this point, you know the drill - just a few signatures to seal the deal." He hands you a sleek pen, his gaze businesslike. "Once this is done, you'll be a stakeholder in one of Megaparsec's most innovative projects to redefine performance testing on Crossroads."`
```

			label ending
			`	The long journey back to the spaceport proceeds in silence, leaving your decision behind to echo through the lives of the people left here.`

mission "Syndicate Business 6"
	name "Visit Howard on <planet>"
	description "Howard wants to tell you about yet another investment opportunity."
	minor
	to offer
		has "Syndicate Business 5: done"
		has "Syndicate Business 5.1 A 10m Expansion Update: offered"
		has "event: syndicate business 10m investment done"
		random < 50
		or
			credits > 300000000
			"net worth" > 600000000
	source
		government "Syndicate"

> 		attributes "core"

Author: Removed

		not planet "Sunracer"
	destination "Sunracer"
	on offer
		conversation
			branch deferred
				has "Syndicate Business 6: deferred"
			`After an uneventful walk through the spaceport, you return to <ship> to see a message from Howard, the Megaparsec salesman who sold you two investments. The message says:`
				goto message
			label deferred

> 			`	After an uneventful trip through the spaceport, you remember that you saved a message from Howard, the Megaparsec salesman who sold you two investments. The message says:`

Author: Hmmm, you didn't buy shares of the business (the first was a bond, and the second was in a fueling depot), buy how's something like this, where you "invested with" Howard? 
```suggestion
			`	After an uneventful trip through the spaceport, you remember saving a message from Howard, the Megaparsec salesman you've already invested with twice. The message says:`
```

> 			`	After an uneventful trip through the spaceport, you remember that you saved a message from Howard, the Megaparsec salesman who sold you two investments. The message says:`

Reviewer 3: That works better, although it's still a little awkward.

			label message
			branch meanopening
				"syndicate business: mean to Howard" > 0
			`	"Dear <first>,`
			`		"I trust that all is well with you. I am pleased to share that the investment opportunity I mentioned in my last message has now arrived. I truly value our partnership and the trust you've placed in me, and I'm confident this next venture will be a great fit for your ambitions.`
			`		"Please meet me on Sunracer at your earliest convenience so we can review the details together. I look forward to catching up and discussing our next steps."`
			`		Howard Vendeur`

> 			`		Investor Relations Manager, Megaparsec Incorporated`

Reviewer 2: Didn't previous missions have this indented further, or am I just misremembering?

Author: Good catch - a couple of Howard's messages had incorrect indenting in this one. Now fixed.

			`		Howard Vendeur`
			`		Investor Relations Manager, Megaparsec Incorporated`
				accept
	on defer
		set "Syndicate Business 6: deferred"
	on accept
		clear "Syndicate Business 6: deferred"
	on complete
		conversation
			`Howard sends you a reminder of his address near the spaceport as you land your <model> on <planet> - curiously on the ninth floor of the Megaparsec corporate office, rather than the eighth floor you met him on last time. He must still be reserving conference rooms for these meetings.`

mission "Syndicate Business 7"
	name "Kicking a Hornet's Nest"
	description `Howard needs you to upload "evidence" to an automated Quicksilver in <waypoints>, destroy it, and meet up on <destination> by <date>.`
	priority
	to offer
		has "Syndicate Business 6: done"
	source "Sunracer"
	waypoint "Oblate"
	destination "Canyon"
	deadline 7
	on offer
		conversation
			`After a short walk through the bustling spaceport, you reach Megaparsec's sky-scraping corporate office. The sleek and modern lobby evokes the fast, light ships they're known for, with polished surfaces, elegant furniture, and subtle lighting coming together seamlessly.`

> 			`	The receptionist greets you with a nod and a warm smile, either remembering you from your prior visits or merely being polite. The elevator ride up to the ninth floor is smooth, the soft hum of machinery fading away as you walk through the elevator doors. Howard's office is modern yet modest, featuring a central smart desk, a few sleek chairs, and large windows overlooking the busy shipyards. Howard himself is standing at one of the expansive windows, looking out at Megaparsec's shipyards below.`

Reviewer 2: Merely changing this last sentence to an active version
```suggestion
			`	The receptionist greets you with a nod and a warm smile, either remembering you from your prior visits or merely being polite. The elevator ride up to the ninth floor is smooth, the soft hum of machinery fading away as you walk through the elevator doors. Howard's office is modern yet modest, featuring a central smart desk, a few sleek chairs, and large windows overlooking the busy shipyards. Howard himself stands at one of the expansive windows, looking out at Megaparsec's shipyards below.`
```

			branch meanhello
				"syndicate business: mean to Howard" > 0
			`	"Glad you could make it," he says, his voice cordial and confident. "This investment opportunity for you is ten times larger than our last deal - but first I need a small favor."`
				goto restofhello
			label meanhello
			`	Howard's gaze doesn't shift from the glass as he says, "You're here. Good. This investment opportunity is ten times larger than the last one - but first I need a small favor."`
			label restofhello
			`	He turns to face you. "We're on the cusp of greatness."`
			choice
				`	"You didn't mention any favor in your message."`
				`	"Am I going to have to jump through hoops for every investment you offer?"`
				`	"The cusp of greatness?"`
					goto great
				`	"That's a bit dramatic with the window, no?"`
					goto great
			`	Howard raises an amused eyebrow and remarks, "Relax, it's nothing you can't handle. Just a minor detour on this road to greatness.`
			label great

> 			`	He turns back towards the window, "Greatness. My own corner office, a luxurious company ship, not to mention a sizable bonus. Director... all I need from you is to help nudge a small event in our favor."`

Reviewer 2: ```suggestion
			`	He turns back towards the window. "Greatness. My own corner office, a luxurious company ship, not to mention a sizable bonus. Director... all I need from you is to help nudge a small event in our favor."`
```

			choice
				`	"What kind of nudge?"`
					goto nextsteps
				`	"The 'greatness' you're talking about is just another promotion?"`
			`	Yet again he briefly turns from the window to face you.`
			`	"Of course.`
			label nextsteps

> 			`	"Said nudge entails a certain degree of discretion and precision. You've demonstrated that during our Marauder acquisition, now we need it for an excision. I need you to plant some evidence onto a ship in <waypoints> to bait the pirates there into attacking a research convoy. Then, regroup on <planet> before the convoy comes through next week, and that should definitively shut down the Marauder project. After the anticipated response by the pirates, we confirm on <planet> that there's no loose ends.`

Reviewer 2: ```suggestion
			`	"Said nudge entails a certain degree of discretion and precision. You've demonstrated that during our Marauder acquisition, now we need it for an excision. I need you to plant some evidence onto a ship in <waypoints> to bait the pirates there into attacking a research convoy. Then, regroup on <planet> before the convoy comes through next week, and that should definitively shut down the Marauder project. After the anticipated response by the pirates, we confirm on <planet> that there's no loose ends."`
```

			`	Howard continues to watch the shipyards below for a long moment before turning from the window with a deliberate finality, and strides to his desk at the center of the room. Planting both hands firmly on the smooth surface, he meets your gaze with an unusual intensity and says, "I need you to kick a hornets' nest and to make sure those hornets sting."`
			choice
				`	"Stinging hornets and nest kicking. You got it."`
					goto restofplan
				`	"Last time I helped you start your Marauder project, now you want me to end it? What's going on, Howard?"`
			branch meanplan
				"syndicate business: mean to Howard" > 0
			`	Howard exhales, shaking his head with a knowing smile. "Starting the Marauder project got me here, but keeping it alive? That won't help me get to the next level." Howard leans casually against the desk, his tone becoming more familiar but still businesslike. "The engineers are whispering that mass production doesn't look feasible. That's bad news for me if the higher-ups catch wind of it before we can control the narrative."`
				goto restofplan
			label meanplan

> 			`	Howard exhales sharply as his eyes narrow in irritation. "Starting the Marauder project served its purpose, but now? It's a liability." At the last word, his gaze flicks over you, cold and quick. "Some engineers don't think that mass production of Marauder ships will be viable. That bodes poorly if the higher-ups catch wind of it before I can control the narrative."`

Author: How's this? Slightly less overtly implying the player is the liability, and speaking about the reasons in more general terms.  
```suggestion
			`	Howard exhales sharply as his eyes narrow in irritation. "Starting the Marauder project served its purpose, but now? It's a liability." At the last word, his gaze flicks over you. "Let's just say it's run its course; I need it shut down before anyone starts asking questions that I can't answer."`
```

			label restofcloser
			choice
				`	"Sounds good."`
					goto askitself
				`	"This all seems a bit more... cutthroat than before."`
				`	"This is too underhanded for me. I'm not working with you anymore."`
					decline
			branch meancutthroat
				"syndicate business: mean to Howard" > 0
			`	Howard's eyes soften as he meets your gaze, a fond smile playing on his lips. "Cutthroat? How far we've come since you saved me from that pirate in Moktar. Every hard move has been a stepping stone as we climb to the peak of this ruthless world. And when I reach the top, I won't forget the loyalty that got me there.`
				goto goldenjewelaround
			label meancutthroat
			`	"Me? Cutthroat?" Howard maintains a hard gaze as he gives you an exaggerated wink, "Don't you forget it. I appreciate what you did on Moktar, but business is business. It's a ruthless world, and all we can do is climb to the top or be trampled beneath." Begrudgingly, he adds, "And I don't forget those who help me get there.`
			label goldenjewelaround
			branch piratearound

> 				"syndicate business golden jewel dead"

Reviewer 2: ```suggestion
				has "syndicate business golden jewel dead"
```
I also just realized that the first mission sets the condition with a colon, but everywhere else it's used without. One of those should be changed, and I think putting a colon in would be better - the first missions are more likely to have been offered then the later ones.

Author: Colons added

				accept
	on enter "Moktar"
		"reputation: Independent (Killable)" = -1000
	npc
		government "Independent (Killable)"
		personality heroic staying vindictive
			confusion 20
		system "Moktar"
		ship "Vanguard" "Golden Jewel"
		to despawn
			has "syndicate business golden jewel dead"
		on encounter
			conversation
				`When you enter Moktar, your sensors detect a familiar pirate Vanguard. Your ship is immediately hailed, and the pirate opens communication.`
				`	"If it isn't a pitiful, perplexing parasite, popping up to plague my peace once more!"`
				choice
					`	"We can't keep meeting like this."`
					`	"Uhhh... who are you again?"`
						goto whomst
				`	"Our paths are as persistently painful as the pestilence - your persistent presence has provoked my peevishness!"`
					goto power
				label whomst
				`	"I am the relentless, raging rapscallion who revels in ripping apart feeble, fumbling fiends!"`
				label power

> 				`	You can't help feeling a sense of deja vu as the Vanguard's engines roar to life and you see a half-baker's dozen fixed guns take aim at your ship, ready to strike.`

Author: Whoops, you're right. I do like the phrasing you suggested, but I also really would like the continuity of the deja vu phrase being the same each time you see the ship. 

Do you think it should stay as-is here, or should I change the earlier versions to this suggestion?

> 				`	You can't help feeling a sense of deja vu as the Vanguard's engines roar to life and you see a half-baker's dozen fixed guns take aim at your ship, ready to strike.`

Reviewer 3: I think the earlier ones should probably be changed.

	on enter "Oblate"
		"reputation: Test Dummy" = 10

	npc assist kill
		government "Test Dummy"
		personality derelict disables heroic staying target
			confusion 120
		system "Oblate"
		ship "Quicksilver Test Dummy" "Syndicate Test Vessel"
		on encounter
			conversation
				`You receive a message from Howard almost immediately upon entering the system:`
				`	"Alright, you're here. Get aboard the Quicksilver to upload the file, make sure it doesn't leave the system in one piece, and finally meet back on <destination>. Remember, no loose ends here."`
				`		Howard Vendeur`
				`		Investor Relations Manager, Megaparsec Incorporated`
		on assist
			dialog "You step into the Syndicate Test Vessel, eerily quiet, the empty ship amplifying every footstep. You hurry to upload Howard's evidence file to the ship. The file uploads in seconds, and you return to the <ship> to leave this ghostly vessel behind."
		on kill
			dialog "With the Quicksilver destroyed, it's time to get to <destination>."

	npc
		government "Pirate"
		personality nemesis staying
		system "Oblate"
		ship "Manta" "Rum Ray"
		on kill

> 			dialog "Consider this an open tab with the Rumrunners - the next round is on you, and we collect in blood!"

Reviewer 2: ```suggestion
			dialog `You receive a brief hail as the Manta explodes: "Consider this an open tab with the Rumrunners - the next round is on you, and we collect in blood!"`
```

			log "Destroyed a Manta called the Rum Ray, causing the Rumrunners to promise vengeance."
	
	npc
		government "Pirate"
		personality nemesis staying
		system "Oblate"
		ship "Wasp" "Swig and Sting"
		ship "Scrapper" "Quick Shot"
		ship "Scrapper" "Bottom of the Barrel"
		
	on visit
		conversation
			`Howard sends you a message as your <model> descends to the planet:`
			`	"What are you doing? Get back up there, board the Quicksilver, and finish the job. We're running out of time and we can't afford mistakes."`
			`		Howard Vendeur`

> 			`		Investor Relations Manager, Megaparsec Incorporated`

Reviewer 2: Optionally, you could add a `launch` node here. It depends on whether Howard has the authority to launch you or not.

Author: I don't think he could, at most he'd just send you the sternly worded message he's already sending.

			branch meancanyonopening
				"syndicate business: mean to Howard" > 0
			`	"Excellent work, Captain. My Shuttle's waiting just outside the spaceport - please meet me there immediately. I appreciate your swift action on this.`
				goto restofcanyonopening
			label meancanyonopening
			`	"Good work. My Shuttle is parked just outside the spaceport - meet me there immediately so we can proceed without further delay. No time to waste."`
			label restofcanyonopening
			`		Howard Vendeur`
			`		Investor Relations Manager, Megaparsec Incorporated`

mission "Syndicate Business 8"
	name "Escort Howard to <planet>"
	description "Escort Howard to <destination> so that he can shore up support for his promotion."
	priority
	to offer
		has "Syndicate Business 7: done"
	source "Canyon"
	destination "Hephaestus"
	on offer
		conversation
			`You step out on the stone field surrounding Canyon's spaceport and quickly locate Howard's Shuttle in an open patch with a wide berth. Climbing aboard, you find Howard waiting, and he greets you with a measured nod.`
			branch meancanyonopening
				"syndicate business: mean to Howard" > 0

> 			`	"Again, outstanding work, <first>", he says with a satisfied tone.`

Reviewer 2: ```suggestion
			`	"Again, outstanding work, <first>," he says with a satisfied tone.`
```

				goto restofcanyonopening
			label meancanyonopening
			`	"Job's done," he states flatly.`
			label restofcanyonopening
			`	"The pirates hit the convoy exactly as planned - the Marauder Project's vital components are all lost, with no loose ends left behind. They'll have no choice but to axe the project, and my bosses don't suspect a thing.`

> 			`	"One last step for that promotion - I need you to escort me to Hephaestus so I can secure a few formal promotion recommendations from Syndicate execs, then we finalize that investment. Ready to go?"`

Reviewer 2: Would Howard be so informal as to call them "execs"?

Author: Swapped to leadership.

			`	"Get it done and we both profit. Ready?"`
			label restofcanyoncloser
			choice
				`	"I'm in."`
					accept
				`	"I'm out."`
			label refused
			action
				"syndicate business: mean to Howard" += 3
			`	Howard stares at you, dumbfounded. After a long moment, he scoffs, then laughs - sharp and nervous. "You're out? You can't be out - not after this! You think Lightspeed won't come for me - won't come for you? I'll tell them any..." Howard pauses, takes a deep breath, and straightens his collar.`
			`	"Look. Just get me to <planet>, I'll lock in that promotion, and back to Sunracer - then we talk investments. You've already done the hard part, don't you want to reap the rewards of your efforts? It's right there."`
			`	He leans in, voice low and persuasive, "You've come this far - don't walk away with the finish line in sight."`
			choice
				`	"Okay, I'll do it."`
					accept
				`	"I told you, we're through. I'm leaving."`
					decline
	npc accompany save
		government "Syndicate"
		personality escort timid
		ship "Shuttle" "Gallant Knight"
		on kill
			dialog "You watch as Howard's Shuttle explodes into a cloud of debris along with any hope of his promotion."
	npc

> 		government "Bounty Hunter"

Reviewer 3: Either could work, but it would probably be better if Howard was attacked. If having the Bounty Hunter label is important, you could always define another government.

> 		government "Bounty Hunter"

Author: Done

			`	"<first> <last>,`
			`	"Promotion secured. Meet me here on <destination> to escort me back to Sunracer - I've got a new ship and office to show you. And of course plans to finalize our nine-figure credit investment."`
			`		Howard Vendeur`
			`		Director of Investor Relations & External Projects, Megaparsec Incorporated`
			choice
				`	(Message back a heartfelt congratulations and that you're on your way.)`
				`	(Respond that nobody cares about his promotion, but you'll be there soon.)`
					goto rude
			`	You immediately receive a response:`
			`	"<first>,`
			`	"Appreciate that. Thank you for the kind words. Looking forward to seeing you soon."`
			`		Howard Vendeur`
			`		Director of Investor Relations & External Projects, Megaparsec Incorporated`
			`	It would seem Howard's responses aren't all automated messages.`
			action
				"syndicate business: mean to Howard" --
			``
				goto end
			label rude
			`	You immediately receive a response:`
			`	"<first>,`
			`	"You tihnk I care what you care abuot? Get here. Now."`
			`		Howard Vendeur`
			`		Director of Investor Relations & External Projects, Megaparsec Incorporated`

> 			`	Howard must be seething to miss typos.`

Reviewer 2: It would be funny, but I'm not sure it would be good to do - it seems an awful lot like a typo. In someone's dialog, it makes sense, but if we use `mis` here we'll probably get bug reports.

> 			`	Howard must be seething to miss typos.`

Reviewer 3: Yeah, I guess that's a good point.

> 			`	Howard must be seething to miss typos.`

Author: Agree here, so going to resolve this comment.

			action
				"syndicate business: mean to Howard" ++
			label end
			``
				accept

	on complete

> 		clear "event: syndicate business Howard schmoozing"

Reviewer 2: You already clear this in the `on offer` section.
```suggestion
```

			`	"Given your last investment, I know that you care about things beyond profits. This is a chance to commit to eco-friendly innovation and safer work practices. Here, we can set new standards for performance and for doing good.`
				goto youin
			label evilsweetner
			`	"Given your last foray into unregulated fuel sources, I know that you're looking for sky-high returns here. There's options here to cut through red tape and to make this facility a real profit multiplier.`
			label youin
			branch upcharge
				"syndicate business: mean to Howard" > 0
			`	"Now, the price for this investment is one hundred million credits for a robust 27,400 credits per day. And just like last time, no need to pull out your wallet - I can still pull a few strings with our friends at the bank and our Finance Division. For you, I've even talked them down to half interest. You'll just need to sign a few documents to get things moving."`
			`	He slides a document over to you, with a sleek pen ready for your signature. "Just sign this first document for the loan and then we can talk through the rest of the documents before signing."`
			choice
				`	(Sign the loan.)`
					goto nextynice

> 				`	"I'm done with these deals, I'm leaving."`

Reviewer 2: ```suggestion
				`	"I'm done with these deals. I'm leaving."`
```

					decline
			label upcharge
			`	Howard's tone carries a hint of impatience. "Now, the price for this investment is one hundred and eleven million credits for a 27,400 credit per day return. I've arranged a loan for you at the bank and through our Finance Division, so you won't need to come up with the investment immediately. I've talked down the loan's interest to be half as high as your prior loan. Just sign these documents to formalize the arrangement."`
			`	He shoves a document over to you, with a flimsy pen ready for your signature. "Just sign this first document for the loan and then we can talk through the rest of the documents before signing."`
			choice
				`	(Sign the loan.)`
					goto nextymean

> 				`	"I'm done with these deals, I'm leaving."`

Reviewer 2: ```suggestion
				`	"I'm done with these deals. I'm leaving."`
```
